src/components/spin-button/spin-button.component.html

```diff
@@ -11,7 +11,8 @@
     <ux-icon name="up" class="spin-button-up-icon"></ux-icon>
 </button>
 
-<input [type]="type"
+<input #input
+       type="number"
```

Comment:
Unfortunately this breaks the timezone spin button in the date picker, so we do need it to be able to support `type="text"`.